step.yml

```diff
@@ -120,9 +120,9 @@ inputs:
   - app_password: ""
     opts:
       title: "Apple ID: Application-specific password"
-      summary: Required if using Apple ID
+      summary: Optional if using Apple ID
       description: |-
-        An application-specific password for the Apple ID.
+        You can overwrite the application-specific password for the Apple ID provided for the connection.
```

Comment:
It seems you can not overwrite the connection's app-specific password: https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/pull/102/files#diff-b1ee36d4a84c1c9e5853aff27c0ab87c33d6ce9fbe2597ff126a071ac2c76488R179
and we also should not allow overwriting it.

This input needs to be used if TFA enabled on the Apple ID, but no app-specific password added to the used Bitrise Apple ID connection.